<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QWidget" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="sign_in">
      <attribute name="title">
       <string>Авторизация</string>
      </attribute>
      <widget class="QLineEdit" name="line_password">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>160</y>
         <width>281</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>MS Reference Sans Serif</family>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::ActionsContextMenu</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Пароль</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="line_login">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>100</y>
         <width>281</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>MS Reference Sans Serif</family>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Логин</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Login_button">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>230</y>
         <width>281</width>
         <height>51</height>
        </rect>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="Button">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>35</red>
             <green>35</green>
             <blue>35</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Highlight">
           <brush brushstyle="SolidPattern">
            <color alpha="170">
             <red>35</red>
             <green>35</green>
             <blue>35</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Link">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>35</red>
             <green>35</green>
             <blue>35</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="Button">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>35</red>
             <green>35</green>
             <blue>35</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Highlight">
           <brush brushstyle="SolidPattern">
            <color alpha="170">
             <red>35</red>
             <green>35</green>
             <blue>35</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Link">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>35</red>
             <green>35</green>
             <blue>35</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="Button">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>35</red>
             <green>35</green>
             <blue>35</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Highlight">
           <brush brushstyle="SolidPattern">
            <color alpha="170">
             <red>35</red>
             <green>35</green>
             <blue>35</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Link">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>35</red>
             <green>35</green>
             <blue>35</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="font">
        <font>
         <family>MS Reference Sans Serif</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="text">
        <string>Войти</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="sign_up">
      <attribute name="title">
       <string>Регистрация</string>
      </attribute>
      <widget class="QLineEdit" name="line_newLogin">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>281</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>MS Reference Sans Serif</family>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Логин</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="line_newPassword">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>90</y>
         <width>281</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>MS Reference Sans Serif</family>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Пароль</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="line_passport">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>150</y>
         <width>281</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>MS Reference Sans Serif</family>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Пасспорт без пробелов</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="line_fullname">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>210</y>
         <width>281</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>MS Reference Sans Serif</family>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>ФИ(О при наличии)</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="line_phone">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>270</y>
         <width>281</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>MS Reference Sans Serif</family>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Телефон в формате +x(x)x-x-x</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="line_email">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>330</y>
         <width>281</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>MS Reference Sans Serif</family>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Электронная почта</string>
       </property>
      </widget>
      <widget class="QPushButton" name="registration_button">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>390</y>
         <width>281</width>
         <height>51</height>
        </rect>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="Button">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>35</red>
             <green>35</green>
             <blue>35</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Highlight">
           <brush brushstyle="SolidPattern">
            <color alpha="170">
             <red>35</red>
             <green>35</green>
             <blue>35</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Link">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>35</red>
             <green>35</green>
             <blue>35</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="Button">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>35</red>
             <green>35</green>
             <blue>35</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Highlight">
           <brush brushstyle="SolidPattern">
            <color alpha="170">
             <red>35</red>
             <green>35</green>
             <blue>35</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Link">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>35</red>
             <green>35</green>
             <blue>35</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="Button">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>35</red>
             <green>35</green>
             <blue>35</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Highlight">
           <brush brushstyle="SolidPattern">
            <color alpha="170">
             <red>35</red>
             <green>35</green>
             <blue>35</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Link">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>35</red>
             <green>35</green>
             <blue>35</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="font">
        <font>
         <family>MS Reference Sans Serif</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="text">
        <string>Зарегистрироваться</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>line_login</tabstop>
  <tabstop>line_password</tabstop>
  <tabstop>Login_button</tabstop>
  <tabstop>line_newLogin</tabstop>
  <tabstop>line_newPassword</tabstop>
  <tabstop>line_passport</tabstop>
  <tabstop>line_fullname</tabstop>
  <tabstop>line_phone</tabstop>
  <tabstop>line_email</tabstop>
  <tabstop>registration_button</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
